certification/formatters/util.go

```diff
@@ -16,7 +18,7 @@ func getResponse(r runtime.Results) UserResponse {
 		for i, check := range r.Passed {
 			passedChecks[i] = checkExecutionInfo{
 				Name:        check.Name(),
-				ElapsedTime: check.ElapsedTime.String(),
+				ElapsedTime: float64(check.ElapsedTime) / float64(time.Millisecond),
```

Comment:
Since check.ElapsedTime is a time.Duration, you can just call check.ElapsedTime.Milliseconds().

Reply:
I tried that but Milliseconds() returns an int and since we wanted a float I didn't use it.

Reply:
We should be able to cast that to float64 like you are doing already no?

`float64(check.ElapsedTime.Milliseconds())`

Reply:
Yes I could do that. The only thing with doing this is we lose some precision after the decimal point but if we don't care about that then it should be fine.

Reply:
@edreyer-1 I'm okay with losing that precision for this use case 👍